common/src/main/java/com/wynntils/features/user/overlays/ShamanTotemTrackingFeature.java

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user.overlays;
+
+import com.mojang.blaze3d.platform.Window;
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.components.Model;
+import com.wynntils.core.components.Models;
+import com.wynntils.core.config.Config;
+import com.wynntils.core.config.ConfigHolder;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.features.overlays.Overlay;
+import com.wynntils.core.features.overlays.OverlayPosition;
+import com.wynntils.core.features.overlays.annotations.OverlayInfo;
+import com.wynntils.core.features.overlays.sizes.GuiScaledOverlaySize;
+import com.wynntils.core.features.properties.FeatureCategory;
+import com.wynntils.core.features.properties.FeatureInfo;
+import com.wynntils.gui.render.FontRenderer;
+import com.wynntils.gui.render.HorizontalAlignment;
+import com.wynntils.gui.render.TextRenderSetting;
+import com.wynntils.gui.render.TextRenderTask;
+import com.wynntils.gui.render.TextShadow;
+import com.wynntils.gui.render.VerticalAlignment;
+import com.wynntils.mc.event.RenderEvent;
+import com.wynntils.mc.utils.McUtils;
+import com.wynntils.utils.StringUtils;
+import com.wynntils.wynn.event.TotemEvent;
+import com.wynntils.wynn.objects.ShamanTotem;
+import java.util.List;
+import net.minecraft.ChatFormatting;
+import net.minecraft.world.entity.decoration.ArmorStand;
+import net.minecraft.world.scores.PlayerTeam;
+import net.minecraft.world.scores.Scoreboard;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+@FeatureInfo(category = FeatureCategory.OVERLAYS)
+public class ShamanTotemTrackingFeature extends UserFeature {
+    @OverlayInfo(renderType = RenderEvent.ElementType.GUI)
+    private final ShamanTotemTimerOverlay shamanTotemTimerOverlay = new ShamanTotemTimerOverlay();
+
+    @Config
+    public boolean highlightShamanTotems = true;
+
+    @Config
+    public static TotemTrackingDetail totemTrackingDetail = TotemTrackingDetail.COORDS;
+
+    @Config
+    public static ChatFormatting firstTotemColor = ChatFormatting.WHITE;
+
+    @Config
+    public static ChatFormatting secondTotemColor = ChatFormatting.AQUA;
+
+    @Config
+    public static ChatFormatting thirdTotemColor = ChatFormatting.RED;
```

Comment:
These configs should be in the overlay itself

Reply:
as well as `TotemTrackingDetail`

Reply:
I can move TotemTrackingDetail and it's enum, but I think xTotemColor should stay since it not only does the config for the overlay, but also the highlights on the entity

Reply:
Oh, well okay, they can stay then